src/components/SignIn.jsx

```diff
@@ -49,17 +57,24 @@ class SignIn extends React.Component {
   render() {
     const { email, password } = this.state;
     return (
-      <form>
+      <form style={sectionStyle}>
+        <div style={{textAlign: "center", paddingTop: "10px"}}>
+          <Typography variant="h3">
+            {"OSSDF DANCEFEST"}
+          </Typography>
+        </div>
         <DialogInput
+          style={{marginTop: "50px"}}
           value={email}
           name="email"
-          label="Email"
+          label={<div style={{color: "white"}}>Email</div>}
```

Comment:
We may just want to make this a function.

`const inputLabel = (textLabel) => ( <div style={{color: "white"}}>{textLabel}</div> );`

`label = {inputLabel("Email" /  "Password")}`